src/javascript/app_2/App/Components/Elements/DenialOfServiceModal/denial_of_service.jsx

```diff
@@ -0,0 +1,39 @@
+import React                from 'react';
+import PropTypes            from 'prop-types';
+import { getAccountOfType } from '_common/base/client_base';
+import { localize }         from '_common/localize';
+import URL                  from '_common/url';
+import FullPageModal        from 'App/Components/Elements/FullPageModal/full_page_modal.jsx';
+import { connect }          from 'Stores/connect';
+
+const onConfirm = (client) => {
+    client.switchAccount(getAccountOfType('virtual').loginid);
+};
+
+const onCancel = () => {
+    window.location.href = URL.urlFor('trading');
+};
+
+const DenialOfServiceModal = ({ client, show }) => (
+    <FullPageModal
+        title={localize('Whoops!')}
+        body={localize('Sorry, Only virtual accounts can access this feature at the moment.')}
+        confirm_button_text={localize('Continue with my virtual account')}
+        cancel_button_text={localize('Visit main website')}
+        onConfirm={() => onConfirm(client)}
+        onCancel={onCancel}
+        show={show}
+    />
+);
+
+DenialOfServiceModal.propTypes = {
+    show: PropTypes.bool,
+};
+
+const denial_of_service = connect(
```

Comment:
@easteregg 
ditto

Reply:
This is a container.

Reply:
So shouldn't be in `Containers` directory?